angularjs/angular.d.ts

```diff
@@ -1738,6 +1738,10 @@ declare namespace angular {
          */
         require?: {[controller: string]: string};
     }
+    
+    interface IComponentBindings {
+		[binding: string]: string;
```

Comment:
There still is a mix of tabs and spaces, as well as some trailing whitespace. The rest of the file uses 4 spaces for indentation. Try turning on whitespace rendering in your editor.